test/test-get-currentexe.c

```diff
@@ -82,5 +82,27 @@ TEST_IMPL(get_currentexe) {
   ASSERT(buffer[0] != '\0');
   ASSERT(buffer[1] == '\0');
 
+  /* Verify uv_exepath is not affected by uv_set_process_title(). */
+  r = uv_set_process_title("foobar");
+  ASSERT(r == 0);
+  size = sizeof(buffer) / sizeof(buffer[0]);
```

Comment:
```suggestion
  size = sizeof(buffer);
```
`sizeof(char)` is always 1.

Reply:
With the exception of the `uv_set_process_title()` call these lines were copied from earlier in the test. I’ll change those as well for consistency.